src/components/Movie.tsx

```diff
@@ -1,16 +1,23 @@
 import { Fragment } from "react";
+import { MovieData } from "../data/types";
 
 import { FavoriteStar } from "./FavoriteStar";
 import styles from "./Movie.module.css";
 
-export function Movie({ className, favorite, movie, toggleFavorite }) {
+export interface MovieProps {
+  favorite: boolean;
+  movie: MovieData;
+  // toggleFavorite: Function; // no can do!
```

Comment:
You can think of `Function` as kind of like the `unknown` for function types. It doesn't know what its parameters or return type are.